src/locales/en.json

```diff
@@ -73,5 +73,11 @@
         }
       ]
     }
+  },
+  "contact": {
+    "headerSection": {
```

Comment:
we have headersection key in the home page also, i know they are not related but i think its better to change the naming here

Reply:
Thanks for reviewing Ibrahim, I renamed the key according to your advice. PTAL